### ISSUE TITLE
Open URL snap package KeePassXC in Ubuntu 14.04 LTS

### ISSUE THREAD

**Issue by wowotiel** [keepassxreboot/keepassxc#507]
## Expected Behavior
open url with the key combination: ctrl+U
choose open url within the context menu

## Current Behavior
open url with the key combination: ctrl+U : nothing happens
choose open url within the context menu : nothing happens

## Possible Solution
Make the key combination: ctrl+U working and also the contextmenu

## Steps to Reproduce (for bugs)
1. Install snap
2. Open or create a database
3. The key combination ctrl+V
4. or click on open url within the context menu

## Context
Without this feature the snap version is not usable for me.
So I am using the program KeePassX 2.03 because in this program the key combination ctrl+U is working ok and also click on open url within the context menu does work

## Your Environment
* KeePassXC version/commit used: 2.1.4 
Revision: cb283bb95ae559d044415f426f841c27bff383f0
* Qt version: Qt 5.6.1
*  libgcrypt 1.7.2-beta
* Extensions: KeepassHTTP and Autotype
* Operating System and version: Ubuntu 14.04 LTS 64-bit

**Comment by ribbons** [comment]
I've hit this issue too (KDE neon User Edition 5.9).

Hope this helps rather than just being noise - If I start `keepassxc` from a terminal and try opening a URL, it prints this error:

```
dbus-send: /snap/keepassxc/9/lib/x86_64-linux-gnu/libdbus-1.so.3: version `LIBDBUS_PRIVATE_1.10.6' not found (required by dbus-send)
```

**Comment by e2jk** [comment]
I hit the same issue on Ubuntu 16.04.2 LTS, keepassxc 2.1.4 installed via Snap.
I did see the same error message about LIBDBUS_PRIVATE on the console yesterday, although it's not shown anymore today.

Note that I'm also hitting the same issue with Ctrl+V for Autotype (issue #506), but that other shortcuts (such as Ctrl+B and Ctrl+C) work fine.

**Comment by wowotiel** [comment]
I have upgraded form Ubuntu 14.04 LTS to Ubuntu 16.04 LTS.
With version 2.2.0 I have tested ctrl+U = open URL.
This function does strangely enough **not** work for me in Ubuntu 16.04 LTS (unlike ctrl+V).

**Comment by e2jk** [comment]
I confirm Ctrl + U doesn't work yet with version 2.2.0 on my Ubuntu 16.04. Running from the command line, the following is shown in the terminal at the moment the shortcut is called:
```dbus-send: /snap/keepassxc/23/lib/x86_64-linux-gnu/libdbus-1.so.3: version `LIBDBUS_PRIVATE_1.10.6' not found (required by dbus-send)```
(same as what @ribbons mentioned in his post above, just updated for the newest version of the `keepassxc` snap)

**Comment by StephenWetzel** [comment]
I confirm I have this same issue on Xubuntu 16.04 and Snap package of KeepassXC 2.2.0.  It works when I use the AppImage, so I'm just using that.

**Comment by droidmonkey** [comment]
This will be fixed in 2.2.1 bring released this weekend
